internal/routes/all_routes.go

```diff
@@ -114,6 +114,10 @@ func apiRouter() http.Handler {
 				r.Get("/", s.ListBuiltinInstanceTypes(ec2_types.InstanceTypesForZone))
 			})
 		})
+
+		r.Route("/feature_flag/{FLAG}", func(r chi.Router) {
```

Comment:
I'd probably not expose it for now, it seems that we can consume other endpoints in the UI, that are already exposed for us, backend should consume feature flags to make it's own decisions only if I understand it correctly.

Reply:
Well I thought so but you wrote this into the ticket. IMHO both UI and API should have its own unleash code checking against the server, no endpoint would ever be needed.

Reply:
yeah, you're right, I was wrong. Let me update the ticket.